NEWS.md

```diff
@@ -5,6 +5,9 @@ main
 --------
 
 
+v0.15.6
+-------
+
 v0.15.5
 -------
 - Integrated land model with snow, soil, canopy
```

Comment:
do we have other changes to include here?

Reply:
In the future it would be great to update the NEWS.md in the PRs that make the changes, but this time I can go back through and see what was merged to update it